test/email-sender.test.js

```diff
@@ -0,0 +1,20 @@
+const emailsender = require('../src/email-sender');
+
+// This tests sendMessage from /src/email-sender.js:
+// It gives a test email and recipient to the sendMessage function
+// and checks if it's resolve returns what is to be expected,
+// which an array of the testRecipient since it returns
+// an array of addresses it sent the email too
+test.skip('Tests if sendMessage resolves with expected info', async () => {
+  // Feeds test data into sendMessage function
+  const testSubjectMessage = 'Test';
+  const testHTML = '<h1>Hello World</h1>';
+  const testRecipient = process.env.NODEMAILER_TESTRECIPIENT;
+  const testReturnValue = await emailsender.sendMessage(
+    testRecipient,
+    testSubjectMessage,
+    testHTML,
+  );
+  // Tests if the expected resolve is correct
+  expect(testReturnValue).resolves.toBe([testRecipient]);
```

Comment:
This tests checks to see if it works given a good email, but we should also add a test with bad data to be more through. I will approve this PR but you should either add one later or create another issue for it.